### ISSUE TITLE
Connection Times out when attempting to create Strapi project with AWS RDS PostgreSQL

### ISSUE THREAD

**Issue by diogoviveiros** [strapi/strapi#10598]
## Bug report

Hello,

I am trying to create a new Strapi project with a PostgreSQL database that I created with Amazon's AWS RDS service. I have set "Public Accessibility" to yes, so I think Strapi should be able to connect. However, I'm always getting a "Connection test failed: connect ETIMEDOUT " error when I to run npx create-strapi-project.

### Steps to reproduce the behavior
Go to CMD.

Run npx create-strapi-project (name of project)

Run the following parameters:

```
`Choose your installation type Custom (manual settings)
? Would you like to use a template? (Templates are Strapi configurations designed for a specific use case) Yes

? Select a template Blog

? Choose your default database client

? Choose your default database client postgres

? Database name: (name of  db found in configurations tab in AWS RDS)

? Database name: (name of db found in configurations tab in AWS RDS)

? Host: (dbname and alphanumerics).us-east-1.rds.amazonaws.com

? Host: (dbname and alphanumerics).us-east-1.rds.amazonaws.com

? Port: (5432)

? Port: 5432

? Username: (name of master user found in configurations tab in AWS RDS)

? Username: (name of master user found in configurations tab in AWS RDS)
? Password: (the password)

? Password: (the password)

? Enable SSL connection: (y/N) y

? Enable SSL connection: Yes`
```

### System

- Node.js version: v14.17.1
- NPM version: 6.14.13
- Strapi version: Haven't instantiated strapi project yet. 
- Database: PostgreSQL (Using AWS Engine PostrgreSQL version 12.5-R1
- Operating system: Whatever AWS runs on their servers, I have no idea

**Comment by kasonde** [comment]
Hi @diogoviveiros, if you haven't seen it already, we have a guide on deploying to AWS available here. With that said, I have a couple of questions, did you make sure to double-check your credentials? You could do this with a tool like pgAdmin/any other tool to see if you can connect with the same credentials you're entering in Strapi. 
For the EC2 server itself, can you confirm that it is in the same region as the RDS instance you've spun up?

**Comment by diogoviveiros** [comment]
Hello, I tried using your tutorial but it was a little confusing simply because I only want to host my database on AWS, not use EC2 (I want to deploy to Heroku separately). Regardless, I just tried connecting to my RDS instance on PGAdmin and it's indeed giving me the same error! I'm not exactly sure what I'm doing wrong.

**Comment by diogoviveiros** [comment]
I've figured it out. Even though I set the instance to be publicly accessible, for some reason I still needed to change the security groups in EC2 to be visible in both IPv4 and IPv6. I did this by pressing the blue "default" VPC link in the Connectivity and Security tab of my instance and then adding all traffic from IPv4 and IPv6 Security Group. Hope this helps somebody in the future who has the same issue

**Comment by kasonde** [comment]
Thanks for sharing the information!
